go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/hashicorp/go-plugin v1.0.1 // indirect
 	github.com/lib/pq v1.1.1 // indirect
 	github.com/mattermost/go-i18n v1.11.0 // indirect
-	github.com/mattermost/mattermost-server v5.12.0+incompatible
+	github.com/mattermost/mattermost-server v1.4.1-0.20190814175728-96e36f73b25d // v5.12.6
```

Comment:
I'm wondering why you are not updating to MM 5.18 right away?

Reply:
Because prior to this PR the plugin was depending on 5.12, and this PR does not introduce anything that would require an upgrade.

Reply:
Just FYI: This would allow removing the `willnorris.com/go/imageproxy` workaround.

Reply:
Yep, I know (from previous PRs).

Reply:
I agree that we should use this chance and clean up - use /v5 and 5.18. This will be going into Jira v2.4, so particular rush AFAIC.